### DISCUSSION TITLE
Add option to exclude default skins from random skin shortcut

### DISCUSSION THREAD

**Discussion by yeahirai** [ppy/osu#30986] [Ideas]
This shortcut is a pretty neat feature of lazer. I personally use it a lot when i'm casually playing just to play around with some of the other skins i have installed. The problem is that i don't have that many skins, in fact there's more default skins on my game than custom ones. This leads to an annoyance where i need to press the shortcut multiple times in order for it to select a skin that i actually use to play. For that reason i think this shortcut could be more useful if we could limit it in this kind of way.

**Comment by N0Y0U2114** [comment]
Roll this in with some more features as a "Random of selected" where you can choose what skins to include in the randomization so if you have multiple versions or edits of skins saved, they can be discluded and only the finished/ones with the edits you prefer are used.
